### ISSUE TITLE
Biome Marker Crash

### ISSUE THREAD

**Issue by ShadowSans55** [FTBTeam/FTB-Presents-Direwolf20-1.12#44]
* **Modpack Version**: 2.2.0

* **Issue**: Game crashes when trying to craft a Biome Marker from Extra Utilities 2 with the autoplace items option from JEI.

* **Link to Log or Crash File Paste**: No crash report it seems.

* **Is it Repeatable?**: Yes.

* **Mod/s Affected**: Extra Utilities 2, JEI, Refined Storage (tried crafting it there).

**Comment by SnowShock35** [comment]
Hello @ShadowSans55 

Thank you for your contribution to the FTB Presents Direwolf20 1.12 mod-pack. Your involvement greatly helps to improve the quality of the product and keep it bug-free and more enjoyable for everyone. Without your input, the pack would cease to be as good as it is. 

I have confirmed that this issue still exists in the latest version of Refined Storage/Extra Utilities/Just Enough Items (JEI). As such, I will report it accordingly to the authors and report back once it has been addressed.

Many thanks once again for your engagement here on the FTBTeam's GitHub.

**SnowShock35 ☃️**

***

> ![SnowShock35](https://cdn.snowshock35.com/avatar-rounded.png)
> ### SnowShock35
> @FTBTeam Pack Developer, Aspiring Full-Stack Engineer, 
> UoN Student, Mod & Modpack Developer, Casual Gamer,
> Nerd, Tea Drinker, Made in the UK 🏳️‍🌈
